rex/rechunk_h5/rechunk_h5.py

```diff
@@ -231,6 +246,154 @@ def time_slice(self):
         """
         return self._time_slice
 
+    @property
+    def rechunk_attrs(self):
+        """
+        Attributes for rechunked files, includes dataset and global attrs
+
+        Returns
+        -------
+        pandas.DataFrame
+        """
+        return self._rechunk_attrs
+
+    @property
+    def global_attrs(self):
+        """
+        Global attributes
+
+        Returns
+        -------
+        pandas.Series
+        """
+        return self._get_attrs('global')
+
+    @property
+    def time_index_attrs(self):
+        """
+        Time index attributes
+
+        Returns
+        -------
+        pandas.Series
+        """
+        return self._get_attrs('time_index')
+
+    @property
+    def meta_attrs(self):
+        """
+        Meta attributes
+
+        Returns
+        -------
+        pandas.Series
+        """
+        return self._get_attrs('meta')
+
+    @property
+    def coordinates_attrs(self):
+        """
+        Coordinates attributes
+
+        Returns
+        -------
+        pandas.Series
+        """
+        return self._get_attrs('coordinates')
+
+    @property
+    def variable_attrs(self):
+        """
+        Variable attributes
+
+        Returns
+        -------
+        pandas.Series
+        """
+        return self._get_attrs('variables')
+
+    @staticmethod
+    def _parse_var_attrs(var_attrs, hub_height=None, version=None):
+        """
+        Parse variable attributes from file if needed
+
+        Parameters
+        ----------
+        var_attrs : str | pandas.DataFrame
+            DataFrame of variable attributes or .json containing variable
+            attributes
+        hub_height : int | None, optional
+            Rechunk specific hub_height, by default None
+        version : str, optional
+            File version number, by default None
+
+        Returns
+        -------
+        var_attrs : pandas.DataFrame
+            DataFrame mapping variable (dataset) name to .h5 attributes
+        """
+        if isinstance(var_attrs, str):
+            var_attrs = pd.read_json(var_attrs)
+        elif not isinstance(var_attrs, pd.DataFrame):
+            msg = ("Variable attributes are expected as a .json file or a "
+                   "pandas DataFrame, but a {} was provided!"
+                   .format(type(var_attrs)))
+            logger.error(msg)
+            raise TypeError(msg)
+
+        var_attrs = var_attrs.where(var_attrs.notnull(), None)
+
+        if version is not None:
+            logger.debug('')
```

Comment:
Lost debug statement?

Reply:
What you haven't heard of the new super concise to the point of cryptic logging methodology ;-)